ui/raidboss/data/02-arr/trial/ifrit-nm.txt

```diff
@@ -24,7 +24,8 @@ hideall "--sync--"
 126.1 "Incinerate"
 
 # 70%
-200.0 "--sync--" GameLog { code: "0044", name: "Ifrit", line: "Succumb to the inferno.*?" } window 200,0
+# en: "Succumb to the Inferno!"
+200.0 "--sync--" BattleTalk2 { npcNameId: "04A1", instanceContentTextId: "0A29" } window 200,0
```

Comment:
I think it's worth it to be consistent with the comments for these, pick a standard set and follow it. You've got the timeline entries here quoted but with no link to XIVAPI, and the trigger file entry for t8 has no quotes but does have a link.

To me just the text is sufficient, e.g. `# en: "Succumb to the Inferno!"`, but I'm fine with having the link too, or dropping the quotes, as long as we're consistent about it.

Reply:
I realize as well that you may not have been able to find this data set in XIVAPI, it's there but slightly different name:
<https://xivapi.com/InstanceContentTextData/2601>

Reply:
In #83, I think I landed on an xivapi link and an en: string with no quotes (mirroring what was already in PotD), so I'll stick with that.